### DISCUSSION TITLE
执行start.bat报错（1.4.1版本）

### DISCUSSION THREAD

**Discussion by wszyquan** [fishaudio/fish-speech#630] [General]
1、github下载源码，执行install_env.bat，执行漫长等待完成。windows11，笔记本
2、执行start.bat。（从网上下载打包下载好的程序1.2可以直接执行，没有任何错误）

报错
===============================================
Active code page: 65001
"HF_ENDPOINT: https://hf-mirror.com"
"NO_PROXY: localhost, 127.0.0.1, 0.0.0.0"
model.pth 已存在，跳过下载。
README.md 已存在，跳过下载。
special_tokens_map.json 已存在，跳过下载。
tokenizer_config.json 已存在，跳过下载。
tokenizer.json 已存在，跳过下载。
config.json 已存在，跳过下载。
firefly-gan-vq-fsq-8x1024-21hz-generator.pth 已存在，跳过下载。
ffmpeg.exe 已存在，跳过下载。
ffprobe.exe 已存在，跳过下载。
asr-label-win-x64.exe 已存在，跳过下载。
Debug: flags = --listen 0.0.0.0:8080 --llama-checkpoint-path "checkpoints/fish-speech-1.4" --decoder-checkpoint-path "checkpoints/fish-speech-1.4/firefly-gan-vq-fsq-8x1024-21hz-generator.pth" --decoder-config-name firefly_gan_vq

Next launch the page...
['', 'D:\\git\\fish-speech-1.4.1\\fish_speech\\webui', 'D:\\git\\fish-speech-1.4.1', 'D:\\git\\fish-speech-1.4.1\\fishenv\\env\\python310.zip', 'D:\\git\\fish-speech-1.4.1\\fishenv\\env\\DLLs', 'D:\\git\\fish-speech-1.4.1\\fishenv\\env\\lib', 'D:\\git\\fish-speech-1.4.1\\fishenv\\env', 'D:\\git\\fish-speech-1.4.1\\fishenv\\env\\lib\\site-packages', '__editable__.fish_speech-0.1.0.finder.__path_hook__']
You are in  D:\git\fish-speech-1.4.1
Traceback (most recent call last):
  File "D:\git\fish-speech-1.4.1\fish_speech\webui\manage.py", line 38, in <module>
    seafoam = Seafoam()
  File "D:\git\fish-speech-1.4.1\fish_speech\webui\launch_utils.py", line 107, in __init__
    super().set(
TypeError: Base.set() got an unexpected keyword argument 'button_shadow'
Press any key to continue . . .

**Comment by AnyaCoder** [comment]
别用release，用最新的clone源码
